game/rooms/room13/esc/hint.esc

```diff
@@ -0,0 +1,2 @@
+:look
+	say player "I can just click the exits apparently."
```

Comment:
This tabulation will probably make the command not to be run. Remove the tab please ;)

Reply:
No. That's actually not a problem. The regexp for commands support arbitrary whitespace at the start of a command.